<launch>
    <!-- run calc_inertia_for_urdf.py script -->
    
    <!-- change this to /path/to/your/catkin_ws -->
    <env name="MESH_WORKSPACE_PATH" value="/home/junting/franka_ws/"/>
    <env name="GAZEBO_MODEL_PATH" value="$(find instruct_to_policy)/models"/>

    <arg name="file_path" default="$(find instruct_to_policy)/models/cabinet_1076/model.sdf"/>
    
    <node name="calc_inertia" pkg="instruct_to_policy" type="calc_inertia_for_urdf.py" output="screen">
        <param name="file_path" value="$(arg file_path)"/>
    </node>
</launch>